frontend/src/components/SingleCarousel/SingleCarousel.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from "react";
+import { Carousel } from "react-bootstrap";
+import ymim1 from "../../assets/ymim1.png";
+import { Navbar } from "react-bootstrap";
+import "../../index.css";
+import logo from "../Assets/logo.png";
+
+const inline = {
```

Comment:
Please move this styling to the dedicated CSS file for this component.

Reply:
Ok,  It uses the same styling  and class name as the multi-carousel on the main page. And it some reason it preventing me from commiting changes.